src/electron/views/report/unified-details-section.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { css } from '@uifabric/utilities';
+import { CommentIcon } from 'common/icons/comment-icon';
+import { DateIcon } from 'common/icons/date-icon';
+import { UrlIcon } from 'common/icons/url-icon';
+import { NamedFC } from 'common/react/named-fc';
+import * as React from 'react';
+import { DetailsSectionProps } from 'reports/components/report-sections/details-section';
+
+export const UnifiedDetailsSection = NamedFC<DetailsSectionProps>('DetailsSection', props => {
+    const { targetAppInfo, description, scanDate, toUtcString } = props;
```

Comment:
For the android case, do we have anything in `targetAppInfo.name` that would be helpful to display? The code seems to reference `device`, but not name

Reply:
I'm a bit confused as well since your screenshot has the device name and the app name?

Reply:
So it turns out when we connect we're concating the device name with the app name and storing the whole thing as connectedDevice. I changed it so TargetAppInfo stores the device name from UnifiedScanResultStore and we concat that with the app name here instead (using connectedDevice from the DeviceStore was causing other problems)